spring-cloud-task-core/src/test/java/org/springframework/cloud/task/util/TestDefaultConfiguration.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *  * Copyright 2015 the original author or authors.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *      http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package org.springframework.cloud.task.util;
+
+import org.aspectj.lang.JoinPoint;
```

Comment:
Do we need this explicit dependency on AspectJ or can we accomplish the same thing with only Spring classes?